renovate.json

```
@@ -23,32 +21,75 @@
       ],
       "enabled": true,
       "versioning": "redhat",
-      "allowedVersions": "/^9(\\.|$)/"
+      "allowedVersions": "/^9(\\.|$)/",
+      "schedule": [
+        "after 5am on tuesday"
+      ]
     },
     {
-      "description": "Keep Go toolset on minor version 1.22 only in OpenShift files",
+      "description": "Keep Go toolset on minor version 1.22 in Containerfile only.",
       "matchManagers": ["dockerfile"],
       "matchFileNames": [
-        "Containerfile.aws-load-balancer-controller",
-        "Dockerfile.openshift",
-        "drift-cache/Dockerfile.openshift"
+        "Containerfile.aws-load-balancer-controller"
       ],
       "matchDatasources": ["docker"],
       "matchPackageNames": [
         "registry.access.redhat.com/ubi9/go-toolset"
       ],
       "enabled": true,
       "versioning": "redhat",
-      "allowedVersions": "/^1\\.22(\\.|$)/"
-    },
-    {
-      "description": "Disable regular Go module updates, only allow vulnerability alerts",
-      "matchManagers": ["gomod"],
-      "enabled": false
+      "allowedVersions": "/^1\\.22(\\.|$)/",
+      "schedule": [
+        "after 5am on tuesday"
+      ]
     }
   ],
-  "vulnerabilityAlerts": {
-    "enabled": true
-  },
-  "osvVulnerabilityAlerts": true
+  "tekton": {
+    "managerFilePatterns": [
+      "/\\.yaml$/",
+      "/\\.yml$/"
+    ],
+    "includePaths": [
+      ".tekton/**"
+    ],
+    "packageRules": [
+      {
+        "matchPackageNames": [
+          "/^quay.io/redhat-appstudio-tekton-catalog//",
+          "/^quay.io/konflux-ci/tekton-catalog//",
+          "/^quay.io/konflux-ci/konflux-vanguard//"
+        ],
+        "enabled": true,
+        "groupName": "Konflux references",
+        "branchPrefix": "konflux/references/",
+        "additionalBranchPrefix": "",
+        "group": {
+          "branchTopic": "{{{baseBranch}}}",
+          "commitMessageTopic": "{{{groupName}}}"
+        },
+        "commitMessageTopic": "Konflux references",
```

Can we have a carry patch prefix?
```suggestion
        "commitMessageTopic": "UPSTREAM <carry>: Konflux references",
```

---

Right, I just did a dry-run with `commitMessageTopic": "Konflux references",` and it correctly adds the prefix from the broader config that we have. I reverted this line back to this version.

Fragment from the dry-run:
```bash
       "branchesInformation": [
         {
           "branchName": "konflux/references/main",
           "prNo": null,
           "prTitle": "UPSTREAM: <carry>: Update Konflux references",
           "result": "not-scheduled",
           "upgrades": [ ...
```

Interestingly when I tested with `"commitMessageTopic": "UPSTREAM <carry>: Konflux references",` it would not duplicate that prefix.

The dry run only shows the PR titles but looks like they are the same as commit messages.